cmd/exchange_wrapper_issues/main.go

```diff
@@ -883,3 +891,22 @@ func outputToConsole(exchangeResponses []ExchangeResponses) {
 		log.Println()
 	}
 }
+
+// distruptFormatting adds in an unused delimiter and strange casing features to
+// ensure format currency pair is used througout the code base.
+func distruptFormatting(p currency.Pair) (currency.Pair, error) {
+	base := p.Base.String()
+	if base == "" {
+		return currency.Pair{}, errors.New("cannot distrupt formatting as base is not populated")
+	}
+	quote := p.Quote.String()
+	if quote == "" {
+		return currency.Pair{}, errors.New("cannot distrupt formatting as quote is not populated")
+	}
+
+	return currency.Pair{
+		Base:      p.Base.Upper(),
+		Quote:     p.Quote.Lower(),
+		Delimiter: "---TEST DELIMITER---",
```

Comment:
I like this for testing of the wrapper to ensure that everything meets the criteria of formatting, but I do not like it showing up like it does in the report.html
![image](https://user-images.githubusercontent.com/9261323/96820753-559b4b00-1472-11eb-909c-a8111d33d4f1.png)

Further, this has broken functionality of the report. You cannot expand any panel like you can on master. I cannot inspect whether this change is working as a result
![image](https://user-images.githubusercontent.com/9261323/96821376-77490200-1473-11eb-96ac-23c837352ceb.png)

Likely a result of the space, but yeah if you're doing this, then please update the formatting of the report to look nicer and have the functionality still work